tests/functional/conftest.py

```diff
@@ -203,6 +214,23 @@ def gl(gitlab_config):
     return instance
 
 
+def is_gitlab_ee(gl: gitlab.Gitlab) -> bool:
+    """Determine if we are running with GitLab EE as opposed to GitLab CE"""
+    try:
+        license = gl.get_license()
+    except gitlab.exceptions.GitlabLicenseError:
+        license = None
+    # If we have a license then we assume we are running on GitLab EE
+    if license:
+        return True
+    return False
```

Comment:
Unless mypy is giving us a headache again here, I'd say this would be a bit simpler to understand/flow:

```suggestion
    try:
        license = gl.get_license()
    except gitlab.exceptions.GitlabLicenseError:
        return False
    # If we have a license then we assume we are running on GitLab EE
    return True
```

Reply:
I'm not sure based on: https://docs.gitlab.com/ee/api/license.html

As the lack of an exception doesn't appear to mean we have a license.

Returns:

    200 OK with response containing the licenses in JSON format. This is an empty JSON array if there are no licenses.

Reply:
Ah I see ok! Although without a license the tests we just enabled would fail, so maybe later we can introduce `core`, `ee`, `premium`, `ultimate` markers.

That's for later when we make the functional suite good enough that it can be run as an acceptance tester for any GitLab instance :grin: